ibis_substrait/tests/compiler/snapshots/test_tpch/test_compile/tpc_h07/tpc_h07.json

```diff
@@ -29,48 +25,27 @@
       "extensionFunction": {
         "extensionUriReference": 2,
         "functionAnchor": 2,
-        "name": "extract:date"
-      }
-    },
-    {
-      "extensionFunction": {
-        "extensionUriReference": 3,
-        "functionAnchor": 3,
-        "name": "multiply:dec_dec"
-      }
-    },
-    {
-      "extensionFunction": {
-        "extensionUriReference": 3,
-        "functionAnchor": 4,
-        "name": "subtract:dec_dec"
-      }
```

Comment:
these scalar functions disappearing is a bit odd -- I'll have to dig in deeper (in a follow-up) to make sure this is working correctly